cypress/e2e/HazardCurves.cy.ts

```diff
@@ -16,10 +16,32 @@ describe("Hazard Curves", () => {
     cy.get("button").contains("Accept").click();
   });
 
-  it("Displays inital charts when first visiting page", () => {
+  it("Displays initial charts when first visiting page", () => {
     cy.get('[role="curve"]').should("have.length", 10);
   });
 
+  it.skip("should call window.print when the print button is clicked", () => {
```

Comment:
why did you add a skipped test?

Reply:
Ahh yes @voj I made some comments about this, It SHOULD work, but I couldn't figure out why it doesn't. And I don't want to drop it, but can't block the release while I figure it out.